Makefile

```diff
@@ -40,6 +40,9 @@ check-licenses:
 	cd functions/go && $(MAKE) check-licenses
 	cd functions/contrib/ts && $(MAKE) check-licenses
 
+check-metadata:
+	scripts/check-metadata.py
```

Comment:
"metadata" by itself is too generic, quality what metadata this refers in the target name and scriptname